frontends/bmd/src/app_end_to_end.test.tsx

```diff
@@ -432,4 +439,41 @@ it('MarkAndPrint end-to-end flow', async () => {
   card.insertCard(makeSuperadminCard());
   await authenticateAdminCard();
   await screen.findByText('Reboot from USB');
+  card.removeCard();
+  await advanceTimersAndPromises();
+
+  // ---------------
+
+  // Configure with Admin card
+  card.insertCard(adminCard, electionDefinition.electionData);
+  await authenticateAdminCard();
+  userEvent.click(
+    screen.getByRole('button', { name: 'Load Election Definition' })
+  );
+  await screen.findByText('Election definition is loaded.');
+  card.removeCard();
+  await advanceTimersAndPromises();
+
+  // Unconfigure with Super Admin card
+  card.insertCard(makeSuperadminCard());
+  await authenticateAdminCard();
+  userEvent.click(screen.getByRole('button', { name: 'Unconfigure Machine' }));
+  const modal = await screen.findByRole('alertdialog');
+  userEvent.click(
+    within(modal).getByRole('button', {
+      name: 'Yes, Delete Election Data',
+    })
+  );
+  await waitFor(
+    () => expect(screen.queryByRole('alertdialog')).not.toBeInTheDocument(),
+    { timeout: 2000 }
```

Comment:
I tried to use timer advances instead but found those to be finicky for this case. As a reminder, this increased timeout was necessary because of [this intentional delay mechanism](https://github.com/votingworks/vxsuite/pull/2236/commits/2e3b18cc2ea58d53da67516fb5b537716eab1e52#diff-e13d32b40c63ca62c4b4f64c0cb80c6fdba9b312cf63eaf4ad7d378a27b0e093R13-R18)